client/account-recovery/reset-password/reset-password-confirm-form/style.scss

```diff
@@ -0,0 +1,50 @@
+.reset-password-confirm-form__title {
+	font-size: 20px;
+	font-weight: 600;
+	margin-bottom: 16px;
+}
+
+.form-button.reset-password-confirm-form__button {
```

Comment:
@folletto I would double check this but its possible that he's using a more specific selector here so that we don't have to use `!important` to override the margin/width.